app/models/leaderboard_row.rb

```diff
@@ -44,6 +49,15 @@ def weak_side_wins
     @corp_wins > @runner_wins ? @runner_wins : @corp_wins
   end
 
+  def add_stat(key, value)
+    @extra[key] = value
+  end
+
+  def method_missing(name)
+    return @extra[name] if @extra.has_key?(name)
+    super
+  end
+
```

Comment:
Strength of schedule seems like a core domain concern: I can mention it to anyone on the NetRunner tourney scene and they'll know roughly what is even if they don't know how to calculate it.

As such, shouldn't `strength_of_schedule` be an explicit field instead of an "extra" field implemented via `add_stat` and `method_missing`?

Just wonderin'.